js/src/components/shipping-rate-section/estimated-shipping-rates-card/typedefs.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * @typedef { import(".~/data/actions").CountryCode } CountryCode
+ */
+
+/**
+ * @typedef {Object} ShippingRateGroup
+ * @property {Array<CountryCode>} countries Array of selected country codes.
+ * @property {string} method Shipping method, e.g. "flat_rate".
+ * @property {string} currency Currency.
+ * @property {number} rate Rate value.
```

Comment:
Updated in [`f332ef1` (#1424)](https://github.com/woocommerce/google-listings-and-ads/pull/1424/commits/f332ef13cf18055fbe5d7186ead6991f8000127d). I've changed it to `undefined` instead of `null`, to make it consistent with minimum order side (where the minimum order's `threshold` can be `undefined`).